en/04_Changelogs/5.0.0.md

```diff
@@ -26,9 +35,67 @@ title: 5.0.0 (unreleased)
   - [Other features](#other-features)
 - [Bugfixes](#bugfixes)
 
+## Change to supported modules {#supported-modules}
+
+Some Silverstripe CMS modules are commercially supported. Silverstripe commits to looking after those modules for the duration of the Silverstripe CMS 5 lifecycle.
+
+Review the [list Commercially Supported Modules for Silverstripe CMS 5](/Project_Governance/Supported_Modules).
+
+### Module not supported in Silverstripe CMS 5 {#unsupported-modules}
+Some modules that were supported in Silverstripe CMS 4 will not be supported in Silverstripe CMS 5. 
+
+Just because a module is not supported officially, doesn't mean that you shouldn't be using it. Unsupported modules are maintained on a "best-effort" basis. You should take this into consideration when choosing to install an unsupported module in your project.
```

Comment:
Not sure how to better phrase this but 'unsupported module' sounds worse than 'module not actively supported' or the like. To me, it somehow implies bigger incompatibility rather than just a lack of official support or a guarantee security issues or bugs will be fixed in a reasonable timeframe. Don't want to stretch us too thin on wording like this, we could be going over it forever, but I can easily imagine some white shirt somewhere reading the changelog and thinking ah, we shouldn't use unsupported modules, heard that from security... 
Maybe we need to always use the long version "modules not supported by Silverstripe within the silverstripe namespace" (which is also not always true, e.g. cwp namespace). I think you get the gist of what I'm trying to convey here.

Reply:
Maybe "community supported" ... which contrast nicely with "commercially supported".

Reply:
I'll updated that header and the content below it to say "community supported"

Reply:
This doesn't seem to have been done?

Reply:
Done now.